### PR TITLE
Adds external wallet info to internals page

### PR DESCRIPTION
**Pull request by NejcZdovc**
Resolves https://github.com/brave/brave-browser/issues/10837

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
- enable rewards
- connect uphold account
- make sure that you see correct status and address on internals page

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: browser/ui/webui/brave_rewards_internals_ui.cc [hunk 1]
```diff
@@ -117,6 +121,13 @@ void RewardsInternalsDOMHandler::RegisterMessages() {
       base::BindRepeating(
           &RewardsInternalsDOMHandler::ClearLog,
           base::Unretained(this)));
+
```
Comment (emerick) [inline browser/ui/webui/brave_rewards_internals_ui.cc:124]: Nit: extra space here.

File: components/brave_rewards/resources/internals/components/externalWallet.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    }
+    case 4: {
+      return getLocale('walletStatusDisconnectedVerified')
+    }
+    case 5: {
+      return getLocale('walletStatusPending')
+    }
+  }
+
+  return 'No wallet'
```
Comment (emerick) [inline components/brave_rewards/resources/internals/components/externalWallet.tsx:36]: This string doesn't require localization?

File: components/brave_rewards/resources/internals/components/externalWallet.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+// Utils
+import { getLocale } from '../../../../common/locale'
+
+interface Props {
+  info: RewardsInternals.ExternalWallet
+}
+
+const getWalletStatus = (status: RewardsInternals.WalletStatus) => {
+  switch (status) {
+    case 0: {
```
Comment (emerick) [inline components/brave_rewards/resources/internals/components/externalWallet.tsx:16]: We can't use the definitions like `WalletStatus.NOT_CONNECTED` here?

File: components/brave_rewards/resources/internals/components/externalWallet.tsx (hunk 2 above)
Comment (NejcZdovc) [reply components/brave_rewards/resources/internals/components/externalWallet.tsx:36]: no, this is just something to print out if we don't have handler for it. Should never be displayed in the UI

File: browser/ui/webui/brave_rewards_internals_ui.cc (hunk 1 above)
Comment (NejcZdovc) [reply browser/ui/webui/brave_rewards_internals_ui.cc:124]: removed

File: components/brave_rewards/resources/internals/components/externalWallet.tsx (hunk 3 above)
Comment (NejcZdovc) [reply components/brave_rewards/resources/internals/components/externalWallet.tsx:16]: sadly no :(

**Pull request merged**